packages/atlas/src/components/_video/VideoPlayer/utils.ts

```diff
@@ -113,6 +114,9 @@ export const hotkeysHandler = (
       }
       return
     case 'KeyC':
+      toggleCaptions()
```

Comment:
playerInstance doesn't have a method for toggling subtitles? Asking out of curiosity?

Reply:
No, you have to do it manually